.eslintrc

```diff
@@ -1,66 +0,0 @@
-{
-  // When adding items to this file please check for effects on sub-directories.
-  "parserOptions": {
-    "ecmaFeatures": {
-      "jsx": true
-    }
-  },
```

Comment:
Removed this bit since we aren't a React project and don't need JSX (based on my very limited knowledge)